drivers/spi/Kconfig.stm32

```diff
@@ -17,10 +17,16 @@ config SPI_STM32_INTERRUPT
 	help
 	  Enable Interrupt support for the SPI Driver of STM32 family.
 
+config SPI_STM32_DMA
+	bool "STM32 MCU SPI DMA Support"
```

Comment:
In I2S driver, which only runs with DMA, enabling I2S selects DMA.
I think we should do the same here, so add:
`select DMA`

Reply:
added